agent/bench-scripts/tests/pbench-fio/test-14.txt

```diff
@@ -27,8 +27,10 @@ The following options are available:
 		file sizes in MiB
 
 	-d str[,str] --targets=str[,str]
-		one or more directories or block devices (default is /tmp/fio)
-		(persistent names for devices highly recommended)
+		one or more files, block devices, or directories (default is /tmp/fio)
+		persistent names for block devices highly recommended; to use a directory,
+		use the /var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/templates/fio-shared-fs.job file or specify a
+		job file with --job-file=<path> which uses the 'directory = $target' syntax
```

Comment:
Since we can guess that the value of `${script_path}/templates/fio-shared-fs.job` is going to be a long path, perhaps we could set up the text so that it lands on a line by itself?